conda/repodata.py

```diff
@@ -0,0 +1,99 @@
+# (c) Continuum Analytics, Inc. / http://continuum.io
+# All Rights Reserved
+#
+# conda is distributed under the terms of the BSD 3-clause license.
+# Consult LICENSE.txt or http://opensource.org/licenses/BSD-3-Clause.
+
+from __future__ import absolute_import, division, print_function, unicode_literals
+from .core import repodata as _impl
+from .connection import CondaSession
+from os.path import join
+
+
+class RepoData(object):
```

Comment:
Yes, I will do the review, but this will take time.
I'm fine with the definition "A RepoData object manages repo-data records." :-)
(To be picky myself: "data" already is plural, thus "datas" is an impossible construct. ;-) )